spec/openapi.yaml

```diff
@@ -68,6 +81,124 @@ components:
         - text
         - firmware
 
+    FileInfo:
+      type: object
+      properties:
+        type:
+          type: string
+          example: "PRINT_FILE"
```

Comment:
This „string“ sounds more like some kind of enum. Do we have a list of allowed values?

Reply:
Good point, should be enum